README/ReleaseNotes/v620/index.md

```diff
@@ -328,6 +328,7 @@ prevent infinite loops or crashes.
 
 ## Build, Configuration and Testing Infrastructure
 
+- Option `python` was renamed to `pyroot`. While still using `python` option, `pyroot` will be enabled authomatically.
```

Comment:
I think the following captures the "automatic" part and mentions the deprecation:

```suggestion
- Option `python` was renamed to `pyroot`. The option spelled `python` is a deprecated alias for `pyroot`.
```